src/action.cpp

```diff
@@ -485,14 +485,15 @@ bool can_butcher_at( const tripoint &p )
 {
     // TODO: unify this with game::butcher
     const int factor = g->u.max_quality( quality_id( "BUTCHER" ) );
+    const int factorD = g->u.max_quality( quality_id( "CUT_FINE" ) );
```

Comment:
Unfortunately this section doesn't yet know what action the player might be performing and must check both. Rather than change the const nature of factor, I decided to add a second variable.